files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -828,196 +792,6 @@
     action: drop
 [[ include "_common" . | indent 2 ]]
 [[ include "_labelingschema" . | indent 2 ]]
-# tokend (missing label)
```

Comment:
Only used one is credentiald https://github.com/giantswarm/credentiald/pull/121#pullrequestreview-1505489855